tests/operator_test.go

```diff
@@ -1434,6 +1382,41 @@ spec:
 			Expect(monitoringLabel).To(Equal("true"))
 		})
 	})
+
+	It("should adopt previously unmanaged entities by updating its metadata", func() {
+		By("removing registration metadata")
```

Comment:
Is there anything worth validating related to this in the "Update" functional test as well?